lib/suspenders/generators/app_generator.rb

```diff
@@ -239,7 +239,12 @@ def remove_routes_comment_lines
 
     def outro
       say 'Congratulations! You just pulled our suspenders.'
-      say "Remember to run 'rails generate airbrake' with your API key."
+      say "Remember to run 'bundle exec honeybadger install' with your API key."
+
+      if options[:heroku]
+        say "Run 'bundle exec honeybadger heroku install' with your API key
+        unless you're using the Heroku Honeybadger add-on"
```

Comment:
Assuming this message is necessary (see above), can you add a period at the end of this?